File: src/test/resources/same/operators/Operators.php
<?php

function andOperator()
{
    if (true && true) {
        echo "";
    }
}

function orOperator()
{
    if (true || true) {
        echo "";
    }
}

function isEqualOperator()
{
    if (true == true) {
        echo "";
    }
}

function isNotEqualOperator()
{
    if (true != true) {
        echo "";
    }
}

function isEqualTypeOperator()
{
    if (true === true) {
        echo "";
    }
}

function isNotEqualTypeOperator()
{
    if (true !== true) {
        echo "";
    }
}

function ternaryOperator()
{
    $a = 10;
    $b = ($a == 1) ? 20 : 30;
}

function declareVariableOperator()
{
    $x = 0;
}

function initializeVariableOperator()
{
    $x = 0;
}

function assignVariableOperator()
{
    $x = 0;
    $x = 1;
}

?>

File: src/test/resources/same/operators/Operators.go
package main

import "fmt"

func andOperator() {
	if true && true {
		fmt.Println("")
	}
}

func orOperator() {
	if true || true {
		fmt.Println("")
	}
}

func isEqualOperator() {
	if true == true {
		fmt.Println("")
	}
}

func isNotEqualOperator() {
	if true != true {
		fmt.Println("")
	}
}

func declareVariableOperator() {
	var x int
}

func initializeVariableOperator() {
	var x int = 0
}

func assignVariableOperator() {
	var x int = 0
	x = 1
}
